cli/dcoscli/service/main.py

```diff
@@ -305,4 +306,8 @@ def _log_marathon(follow, lines, ssh_config_file):
 
     emitter.publish(DefaultError("Running `{}`".format(cmd)))
 
+    # In the tests, we SIGKILL this process group, and the output
+    # doesn't get flushed unless we explicitly do so here.
+    sys.stderr.flush()
```

Comment:
Well, I care about leaky abstractions. I think we want to `flush` for every `emitter.publish`. If we want to make a decision based on performance let's measure it so that it is an informed decision.